frontend/src/components/elements/CodeBlock/CodeBlock.scss

```diff
@@ -21,8 +21,13 @@
   See https://prismjs.com/download.html#themes=prism&languages=markup+css+clike+javascript
 */
 
-div.code-block {
-  margin: 0 0 1rem 0;
+div.stCodeBlock {
```

Comment:
We don't need the `div` right? 

(Or, if it's needed, explain why)

Reply:
It's not needed but it is a good practice if the class it's gonna to be used just to a specific element. It's a more efficient selector, but i can remove it if you want there's no sensitive difference